app/services/item_service.rb

```diff
@@ -0,0 +1,13 @@
+class ItemService
+  class << self
+    def conn
+      Faraday.new(url: 'http://localhost:3000/api/v1/items')
```

Comment:
Make your connection just to the domain. ` Faraday.new(url: 'http://localhost:3000`

`api/v1/items .....` is all part of an endpoint, so this should be getting passed in through the get method that you have on line 8 below.